src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/Example1.java

```diff
@@ -5,19 +5,23 @@
   </module>
 </module>
 */
+//non-compiled with javac: Compilable with Java11
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
 
 // xdoc section -- start
 import java.io.IOException;
 import java.net.URL;
 
-import java.io.IOException; // violation, extra separation before import
-                            // and wrong order, comes before 'java.net.URL'.
+import java.io.IOException;
+// 2 violations above:
+//  'Extra separation in import group before 'java.io.IOException''
+//  'Wrong order for 'java.io.IOException' import.'
 import javax.net.ssl.TrustManager; // violation, extra separation due to above comment
 import javax.swing.JComponent;
 import org.apache.http.conn.ClientConnectionManager; // OK
```

Comment:
please remove "// OK" from all files that you changing.

Reply:
there should be NO comments like "// OK".

the only allowed OK comments are like `// OK, alphabetical case-sensitive ASCII order, 'P' < 'a'` that have extra wording to explain why it is ok. So such comments with details are ok to stay.

Reply:
> please remove "// OK" from all files that you changing.

Done